src/index.html

```diff
@@ -7,12 +7,57 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Stars</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
     <h1>Stars</h1>
+
+    <div class="stars__block stars--1">
+      <span class="star star--active"></span>
+      <span class="star star--active"></span>
+      <span class="star star--empty"></span>
+      <span class="star star--empty"></span>
+      <span class="star star--empty"></span>
+    </div>
```

Comment:
Fix classes everywhere
```suggestion
    <div class="stars stars--1">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```